content/patterns/disclosure/examples/disclosure-navigation.html

```diff
@@ -199,22 +191,22 @@ <h2 id="kbd_label">Keyboard Support</h2>
                 <kbd>Tab</kbd><br>
                 <kbd>Shift + Tab</kbd>
               </th>
-              <td>Move keyboard focus among top-level buttons, and if a dropdown is open, into and through links in the dropdown.</td>
+              <td>トップレベルのボタン間、及びドロップダウンが開いている場合は、ドロップダウン内のリンクを通じてキーボードフォーカスを移動する。</td>
```

Comment:
「ドロップダウン内のリンクを通じて」という訳が、若干わかりにくいかなと感じました。through のニュアンスですが、ここでは「〜を通して」というのはいかがでしょうか？

```suggestion
              <td>トップレベルのボタン間、及びドロップダウンが開いている場合は、ドロップダウン内のリンクを通してキーボードフォーカスを移動する。</td>
```

Reply:
```suggestion
              <td>トップレベルのリンクとボタン間をキーボードフォーカスを移動させる。またドロップダウンが開いている場合は、ドロップダウン内のリンクを通して移動させる。</td>
```